mklove/modules/configure.zlib

```diff
@@ -48,7 +48,7 @@ function install_source {
     echo "### Installing $name $ver from source to $destdir"
     if [[ ! -f Makefile ]]; then
         mkl_download_archive \
-            "https://zlib.net/zlib-${ver}.tar.gz" \
+            "https://zlib.net/fossils/zlib-${ver}.tar.gz" \
```

Comment:
I think we're better upgrading to 1.2.13 since the releases in fossils/ are considered obsolete, but I can fix that in a follow-up PR.

(side note: this is the second time this year that old zlib releases are moved or made unavailable, dependencies are fragile..)

Reply:
Thanks for offering to do the follow-up PR
Personally I'd be tempted to have this bash script try both with and without the `fossils/`